core/unit_test/tools/TestAllCalls.cpp

```diff
@@ -50,19 +50,26 @@
 #include <Kokkos_Core.hpp>
 
 int main() {
+#ifdef KOKKOS_ENABLE_CXX11_DISPATCH_LAMBDA
   Kokkos::initialize();
   {
-    Kokkos::View<int*> src_view("source", 10);
-    Kokkos::View<int*> dst_view("destination", 10);
+    using execution_space = Kokkos::DefaultExecutionSpace;
+    using memory_space    = typename execution_space::memory_space;
+    Kokkos::View<int*, memory_space> src_view("source", 10);
+    Kokkos::View<int*, memory_space> dst_view("destination", 10);
     Kokkos::deep_copy(dst_view, src_view);
     Kokkos::parallel_for(
-        "parallel_for", 1, KOKKOS_LAMBDA(int i) { (void)i; });
+        "parallel_for",
+        Kokkos::RangePolicy<Kokkos::DefaultExecutionSpace>(0, 1),
```

Comment:
You could replace `Kokkos::DefaultExecutionSpace` by `execution_space` here and below if we are already defining the alias.

Reply:
Good catch, I added this prior to the using statement and never went back to cleanup. Thanks!

Reply:
@masterleinad cleaned up like you suggested, reapplied clang-format and rebased for clean commit history. Should be good to go now.

Reply:
Thanks!